kuma/static/styles/components/wiki/contributors.scss

```diff
@@ -17,3 +19,29 @@ contributors block at the bottom of the article
         display: none;
     }
 }
+
+/*
+React contributors block at the bottom of the article
```

Comment:
I'm guessing we don't yet have a framework for separating SCSS/CSS for the wiki vs. the react. David had a "framework" where he'd eye over the JS in the jQuery and just rewrite in these new .jsx files.